### PR TITLE
Add docker publish command

### PR DESCRIPTION
**Pull request by guicaulada**

### CODE REVIEW

File: pipelines/docker.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func GrafanaImageTags(base BaseImage, registry string, opts TarFileOpts) []strin
 		version += "-ubuntu"
 	}
 
-	if opts.Distro != "" && opts.Distro != "linux/amd64" {
```
Comment (guicaulada) [inline pipelines/docker.go:51]: This is done because following https://drone.grafana.net/grafana/grafana-private-mirror/7532/1/6 the -amd64 is expected on image-tags.

**Review by kminehart — APPROVED** [review]
Makes sense to me. I would appreciate more comments in that triple-nested for loop as it was a bit hard to follow but makes sense to me.

Also, I see that we're continuing the pattern of putting things in the `pipelines` package. I think for the next few we should avoid that and try putting them in their own package or something. There's a refactor that we eventually need to do so I think it would be good to not add more to `pipelines` if we can avoid it.
